<launch>

    <arg name="uav_id" default="1"/>
    <arg name="gcs_machine" default="Station1"/>

    <machine name="$(arg gcs_machine)"
             address="localhost" />

   <group ns="UAV$(arg uav_id)" >

        <include file="$(find machine_vision)/launch/node_object_detection.launch" >
            <arg name="machine_name" value="$(arg gcs_machine)" />
        </include>

        <include file="$(find rqt_gcs)/launch/jetson.launch" >
            <arg name="tgt_system" value="$(arg uav_id)" />
            <arg name="uav_id" value="$(arg uav_id)"/>
            <arg name="machine_name" value="$(arg gcs_machine)" />
        </include>

        <!--<include file="$(find rqt_gcs)/launch/gcs.launch"/>-->

   </group>

</launch>
